.github/workflows/build-m1-binaries.yml

```diff
@@ -85,3 +85,68 @@ jobs:
           for pkg in dist/*; do
             aws s3 cp "$pkg" "s3://pytorch/whl/${CHANNEL}/cpu/" --acl public-read
           done
+  build_conda:
+    name: "Build TorchAudio M1 conda packages"
+    runs-on: macos-m1-11
+    strategy:
+      matrix:
+        py_vers: [ "3.8", "3.9", "3.10" ]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Set CHANNEL (only for tagged pushes)
+        if: ${{ github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags/') }}
+        run: |
+          # reference ends with an RC suffix
+          if [[ ${GITHUB_REF_NAME} = *-rc[0-9]* ]]; then
+            echo "CHANNEL=test" >> "$GITHUB_ENV"
+          fi
+      - name: Install conda-build and purge previous artifacts
+        shell: arch -arch arm64 bash {0}
+        run: |
+          . ~/miniconda3/etc/profile.d/conda.sh
+          conda install -yq conda-build
+          conda build purge-all
+      - name: Build TorchAudio M1 conda package
+        shell: arch -arch arm64 bash {0}
+        env:
+          ENV_NAME: conda-env-${{ github.run_id }}
+          PYTHON_VERSION: ${{ matrix.py_vers }}
+          CU_VERSION: cpu
+          USE_FFMPEG: true
+          USE_OPENMP: false
+          USE_CUDA: false
+        run: |
+          . ~/miniconda3/etc/profile.d/conda.sh
+          set -ex
+          . packaging/pkg_helpers.bash
+          if [[ $CHANNEL == "test" ]]; then
+            setup_base_build_version
+          else
+            setup_build_version
+          fi
+          git submodule update --init --recursive
+
+          export SOURCE_ROOT_DIR=$(pwd)
+          export FFMPEG_ROOT="${SOURCE_ROOT_DIR}/third_party/ffmpeg"
+          . packaging/ffmpeg/build.sh
```

Comment:
I missed in the wheel build PR, but we need to build ffmpeg for PyPI distributions as well.

Reply:
Thank you. Done